util/util.go

```diff
@@ -1,9 +1,37 @@
 package util
 
-import "os/exec"
+import (
+	"errors"
+	"net"
+	"os/exec"
+	"strings"
+)
 
 // IsCommandAvailable check command exits.
 func IsCommandAvailable(cmd string) bool {
 	_, err := exec.LookPath(cmd)
 	return err == nil
 }
+
+func GetClientIp() (string, error) {
+	addresses, err := net.InterfaceAddrs()
+
+	if err != nil {
+		return "", err
+	}
+
+	for _, address := range addresses {
+		if ipnet, ok := address.(*net.IPNet); ok && !ipnet.IP.IsLoopback() {
+			if ipnet.IP.To4() != nil {
+				return ipnet.IP.String(), nil
+			}
+
+		}
+	}
+
+	return "", errors.New("can not find the client ip addresses")
+}
+
+func CountWords(s string) int {
+	return len(strings.Fields(s))
+}
```

Comment:
Overall, the code patch seems to be relatively simple and straightforward. However, there are a few potential issues that could be addressed:

1. Error handling: The `IsCommandAvailable` function does not return any error messages if the `exec.LookPath` call fails. It would be better to add some error handling to provide more information about why the function failed.

2. Security: The `GetClientIp` function retrieves all available IP addresses on the machine, which could potentially include private or sensitive addresses. It would be better to filter the results to only return public IP addresses.

3. Performance: The `CountWords` function uses the `strings.Fields` function to split a string into words, which can be inefficient for very long strings. It would be better to use a more optimized algorithm to count the words.

Overall, these are relatively minor issues that could be easily addressed.